sensor_msgs/Image image
---
bool is_detected
int32 detected_loop_closure_id